src/frontends/paddle/src/op/sum.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs sum(const NodeContext& node) {
+    auto data = node.get_ng_inputs("X");
+    const auto input_type = data[0].get_element_type();
+    const auto shape = data[0].get_shape();
```

Comment:
Shall we check the shape of each element in list to make sure they are same ?
You can leverage  'PADDLE_OP_CHECK' to log a message if the shapes are not aligned.

Reply:
updated